docs/guides/implementing-connectors.md

```diff
@@ -0,0 +1,88 @@
+---
+id: implementing-connectors
+title: Implement connectors
+description: "Connectors communicate with any system or technology, reducing the time it takes to automate and orchestrate business processes across systems."
+keywords: [connector, modeling, connectors]
+---
+
+The launch of [Camunda Platform 8](../components/concepts/what-is-camunda-platform-8.md) also introduced an integration framework with a key goal: integrate faster to reduce the time it takes to automate and orchestrate business processes across systems.
+
+This goal is achieved using [connectors](../components/modeler/web-modeler/connectors/index.md). Ready to use out of the box, connectors help automate complex [business processes](../components/concepts/processes.md) by inserting them into [BPMN diagrams](./automating-a-process-using-bpmn.md) within [Web Modeler](../components/modeler/about.md), and configuring them via the properties panel.
```

Comment:
```suggestion
[Connectors](../components/modeler/web-modeler/connectors/index.md) achieve this goal. 
```
Can we flip this? Bit of a nitpick, I know.

Reply:
All about that active voice!